### ISSUE TITLE
Support for long types

### ISSUE THREAD

**Issue by fakedrake** [ennocramer/floskell#47]
Could we get support for

```
newtype MyMonadT a b m = MyMonad { runMyMonad :: StateT ([(a, a -> b)]) (ReaderT a (ExceptT [IM.IntMap b]) (WriterT [IS.IntSet x] m))}
```
to become something more sensible than

```
newtype MyMonadT a b m =
  MyMonad { runMyMonad :: StateT ([(a, a -> b)]) (ReaderT a (ExceptT [IM.IntMap b]) (WriterT [IS.IntSet x] m))
          }
```

for example

```
newtype MyMonadT a b m = MyMonad {
  runMyMonad :: StateT ([(a, a -> b)])
    (ReaderT a
     (ExceptT [IM.IntMap b])
      (WriterT [IS.IntSet x] m))
  }
```
would be nice

**Comment by ennocramer** [comment]
Fixed in `master` as of `592e181d50848510175e495caa91fbfa4fb8a70a`.

**Comment by ennocramer** [comment]
Released with `floskell-0.10.4`.
